# The proxy is a tool which remaps joy messages to desired
# joystick tool topics.
#
# The proxy provides up to 4 remapping topics. You can only select one at a
# time. Alternatively, you can disable the joystick remapping.
#
# Commands:
# - disable: <proxy_button_id> + <disable_button_id>
# - select channel n: <proxy_button_id> + <channel_n_button>
#
# Note: Any message in which the "proxy_button" is pressed, will not be
# forwarded!.
#
# see available mappings on src/bender_joy/xbox.py

# button to enable proxy methods
# (default: XBOX)
b_proxy: 'XBOX'

# disable joystick
# (default: Back)
b_disable: 'BACK'

# 4 channels
# - By default, the proxy remaps to the first button
# - this array must have a size of 4
b_channels: ['A', 'B', 'X', 'Y']


